include/lsst/afw/formatters/Utils.h

```diff
@@ -139,7 +141,24 @@ int extractVisitId(std::shared_ptr<lsst::daf::base::PropertySet const> const& pr
 int extractCcdId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 int extractAmpId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 
+/**
+Format a PropertySet into a single FITS header string
+
+@param[in] prop  Properties to format
+@return a FITS header string (exactly 80 characters per "card", no line terminators)
```

Comment:
It is purely internal. The existing function is *only* used by Wcs and I have a ticket filed to get rid of it once we get rid of that class.